packages/@guardian/source-foundations/src/index.test.ts

```diff
@@ -32,6 +32,7 @@ it('Should have exactly these exports', () => {
 		'body',
 		'bodyObjectStyles',
 		'bodySizes',
+		'bodyString',
```

Comment:
i'm not 100% comfortable with naming things after their type – i think it can more instructive to name things after their _use_, but I don't know what i'd make of `body`, `bodyObjectStyles`, `bodySizes` and  `bodyCSS` etc if i was trying to get some css for styling body text either (or maybe more pertinently, if i was going to use it with emotion, which is probably also a more common use-case).

i don't want to open a larger can of worms here, but an api like `fonts.body.sizes`, `fonts.body.css`, `fonts.body.emotionStyles` seems more explicit to me, but comes with a greater set of changes as well as being untreeshakable :(

given we already have `bodyObjectStyles`, what do people think about:

- `bodyEmotionStyles`
- `bodyObjectStyles`
- `bodySizes`
- `__partialFunctionalityCSSBody` <- this is trying to achieve something similar to `__dangerouslySetInnerHTML`

cc @ob6160 @joecowton1 @tjsilver @SiAdcock